app/_src/guides/restrict-permissions-to-selected-namespaces-on-kubernetes.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Restrict permissions to selected namespaces on Kubernetes
+content_type: tutorial
+---
+
+{% capture docs %}/docs/{{ page.release }}{% endcapture %}
+{% assign Kuma = site.mesh_product_name %}
+{% assign kuma = site.mesh_install_archive_name | default: "kuma" %}
+{% assign kuma-control-plane-workloads = kuma | append: "-control-plane-workloads" %}
+
+By default, {{ Kuma }} deployed on Kubernetes has permissions to observe and react to events from resources across the entire cluster. While this behavior simplifies initial setup and testing, it might be too permissive for production environments. Limiting {{ Kuma }}'s access to only necessary namespaces helps enhance security and prevents potential impact on unrelated applications.
+
+Starting from version 2.11, you can restrict the permissions granted to the {{ Kuma }} control plane. This guide explains how to limit {{ Kuma }} to specific namespaces, giving you greater control over security and resource management.
+
+## Prerequisites
+
+Before you start, ensure you have:
+
+* A clean, running, and accessible Kubernetes cluster
```

Comment:
can we just add step with:
```
minikube start -p mesh-zone
```

as we do in quickstart?

Reply:
addressed